src/css/object/project/jop-search-wrapper.scss

```
@@ -0,0 +1,10 @@
+.p-jop-search-wrapper {
```

こちらもセレクタとファイル名以下になるかと思います！

```suggestion
.p-job-search-wrapper {
```